man/fifelse.Rd

```diff
@@ -15,12 +15,16 @@
 }
 \details{
 In contrast to \code{\link[base]{ifelse}} attributes are copied from the first non-\code{NA} argument to the output. This is useful when returning \code{Date}, \code{factor} or other classes.
+
+\code{fifelse}, contrary to \code{\link[base]{ifelse}}, evalutes both \code{yes} and \code{no} arguments for type checking regardless the result of \code{test}. This means that neither \code{yes} nor \code{no} should be recursive function calls, use \code{fcase} instead.
```

Comment:
evaluates*
regardless of*
semicolon is more appropriate instead of comma in between 'This means that neither yes nor no should be recursive function calls' and 'use fcase instead'
Also 'fifelse, contrary to ifelse, ...' can be rephrased to something simpler like 'Unlike base ifelse, fifelse evaluates ...'